gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/GenericStatement.java

```diff
@@ -596,7 +599,8 @@ private final PreparedStatement prepMinion(
 					if (routeQuery && (
 							INSERT_INTO_TABLE_SELECT_PATTERN.matcher(source).matches() ||
 							PUT_INTO_TABLE_SELECT_PATTERN.matcher(source).matches() ||
-              FUNCTION_DDL_PREFIX.matcher(source).matches() )) {
+                            FUNCTION_DDL_PREFIX.matcher(source).matches() ||
+							ALTER_TABLE_COLUMN.matcher(source).matches())) {
```

Comment:
correct the indentation. The new line added should just start after a couple of spaces from 'FUNCTION_DDL_PREFIX.matcher(source).matches() ||' line. Actually for the existing line too. A gap of 2 spaces for the FUNCTION line and the ALTER line should be good.